new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiConfigMap.java

```diff
@@ -321,25 +346,80 @@ public void testCreateMiiConfigMapDomain() {
     }
 
     int adminServiceNodePort = getServiceNodePort(domainNamespace, adminServerPodName + "-external", "default");
-    oracle.weblogic.kubernetes.utils.ExecResult result = null;
+    ExecResult result = null;
+    checkJdbc = new StringBuffer("status=$(curl --user weblogic:welcome1 ");
+    checkJdbc.append("http://" + K8S_NODEPORT_HOST + ":" + adminServiceNodePort)
+         .append("/management/weblogic/latest/domainConfig")
+         .append("/JDBCSystemResources/TestDataSource/")
+         .append(" --silent --show-error ")
+         .append(" -o /dev/null ")
+         .append(" -w %{http_code});")
+         .append("echo ${status}");
+    logger.info("CheckJdbc: curl command {0}", new String(checkJdbc));
+    try {
+      result = exec(new String(checkJdbc), true);
+    } catch (Exception ex) {
+      logger.info("CheckJdbc: caught unexpected exception {0}", ex);
+      fail("CheckJdbc:  got unexpected exception" + ex);
+    }
+    logger.info("CheckJdbc: curl command returns {0}", result.toString());
+    assertEquals("200", result.stdout(), "DataSource configuration not found");
```

Comment:
If the test data source fails, what http response code do you get?

Reply:
It returns 404